doc/importexport.rst

```diff
@@ -0,0 +1,263 @@
+                    "height": 300,
+                    "marginLeft": 10,
+                    "marginTop": 10,
+                    "showAxes": False,
+                    "projectionDir": (0.5, 0.5, 0.5),
+                    "strokeWidth": 0.25,
+                    "strokeColor": (255, 0, 0),
+                    "hiddenColor": (0, 0, 255),
+                    "showHidden": True,
+                },
+            )
+
+Which results in the following image:
+
+..  image:: _static/importexport/box_custom_options.svg
+
+Exporting STL
+##############
+
+The STL exporter is capable of adjusting the quality of the resulting mesh, and accepts the following parameters.
+
+* ``fileName`` - The path and file name to write the STL output to.
+* ``tolerance`` - A linear deflection setting which limits the distance between a curve and its tessellation. Setting this value too low will result in large meshes that can consume computing resources. Setting the value too high can result in meshes with a level of detail that is too low. Default is 0.1, which is good starting point for a range of cases.
+* ``angularTolerance`` - Angular deflection setting which limits the angle between subsequent segments in a polyline. Default is 0.1.
+
+For more complex objects, some experimentation with ``tolerance`` and ``angularTolerance`` may be required to find the 
+optimum values that will produce an acceptable mesh.
+
+.. code-block:: python
+
+    import cadquery as cq
+    from cadquery import exporters
+
+    result = cq.Workplane().box(10, 10, 10)
+
+    exporters.export(result, '/path/to/file/mesh.stl')
+
+Exporting AMF
+##############
+
+The AMF exporter is capable of adjusting the quality of the resulting mesh, and accepts the following parameters.
+
+* ``fileName`` - The path and file name to write the AMF output to.
+* ``tolerance`` - A linear deflection setting which limits the distance between a curve and its tessellation. Setting this value too low will result in large meshes that can consume computing resources. Setting the value too high can result in meshes with a level of detail that is too low. Default is 0.1, which is good starting point for a range of cases.
+* ``angularTolerance`` - Angular deflection setting which limits the angle between subsequent segments in a polyline. Default is 0.1.
+
+For more complex objects, some experimentation with ``tolerance`` and ``angularTolerance`` may be required to find the 
+optimum values that will produce an acceptable mesh. Note that parameters for AMF color and material are absent.
+
+.. code-block:: python
+
+    import cadquery as cq
+    from cadquery import exporters
+
+    result = cq.Workplane().box(10, 10, 10)
+
+    exporters.export(result, '/path/to/file/mesh.amf', tolerance=0.01, angularTolerance=0.1)
+
+
+Exporting TJS
+##############
+
+The TJS (ThreeJS) exporter is capable of adjusting the quality of the resulting JSON-based mesh, and accepts the following parameters.
```

Comment:
```suggestion
The TJS (ThreeJS) exporter produces a file in JSON format that describes a scene for the ThreeJS WebGL renderer. The objects in the first argument are converted into a mesh and then form the ThreeJS geometry for the scene. The mesh can be adjusted with the following parameters.
```

I think anyway. The terminology used in ThreeJS is significantly different to CAD. Also not helped by ThreeJS V3 JSON model format being retired and these terms changing a lot for the current V4.